battleship.py

```diff
@@ -62,8 +62,14 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    return
-
+    #implemented empty grid function
+    grid=[]
+    for i in range(rows):
+        l1 = []

```

Comment:
write a variable names in such a way that they are readable.

Reply:
i had modified